sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/key_type.hpp

```diff
@@ -17,45 +19,77 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Keys {
    * @brief The JsonWebKey types.
    *
    */
-  enum class JsonWebKeyType
-  {
+  class KeyVaultKeyType {
+  private:
+    std::string m_value;
+
+  public:
+    /**
+     * @brief Construct a new JWT Type object
+     *
+     * @param jwt The JWT as string.
+     */
+    explicit KeyVaultKeyType(std::string jwt) : m_value(std::move(jwt)) {}
```

Comment:
yes,

Reply:
In that case, let's explicitly state the whole name rather than abbreviation to make the documentation clearer for those who don't know the domain
```suggestion
     * @param jwt The JSON Web Token (JWT) as a string.
     */
    explicit KeyVaultKeyType(std::string jwt) : m_value(std::move(jwt)) {}
```